odata-data-aggregation-ext/3.2 Basic Aggregation.md

```diff
@@ -165,7 +165,7 @@ results in
 
 ##### ##subsubsubsubsec Standard Aggregation Method `countdistinct`
 
-The aggregation method `countdistinct` can be applied to arbitrary collections to count the distinct values. Instance comparison uses the definition of equality in [OData-URL, section 5.1.1.1.1](#ODataURL).
+The aggregation method `countdistinct` can be applied to arbitrary collections to count the distinct values. Instance comparison uses the definition of equality in [OData-URL, section 5.1.1.1.1]($$$OData-URL$$$#Equals).
```

Comment:
I'd use section names in cross-spec links, they are usually more stable than the numbers

```suggestion
The aggregation method `countdistinct` can be applied to arbitrary collections to count the distinct values. Instance comparison uses the definition of equality in [OData-URL, section "Equals"]($$$OData-URL$$$#Equals).
```